Nanayeb34 add new tts script

Nanayeb34 opened the pull request:
# Description

Added the optimized TTS competition script



# Reviewers

@wsedor @kojomensahonums @naafidasana

Nanayeb34 committed:
Added TTS competition script

Nanayeb34 committed:
Added TTS competition script

Nanayeb34 committed:
Update README.md

kojomensahonums reviewed (commented):
Are you making the models available with the config and model paths you've set? It looks they also point more to a colab environment 'content/'?

wsedor commented:
@Nanayeb34 this PR has bene out for sometime. If it's no longer needed please close it

Nanayeb34 commented:
@wsedor i think no one has reviewed this PR yet that's why it can't be merged. Could you approve it so i merge it?

wsedor commented on text-to-speech/General_TTS_inference_API_notebook_version2.ipynb:
```diff
@@ -0,0 +1,196 @@
+      "cell_type": "code",
+      "source": [
+        "#Replace comment section below with your auth token\n",
+        "!ngrok config add-authtoken #place your access token here"
```
let's switch this to import the token from the secrets section. Similar to the change in https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R24 and https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R131

Nanayeb34 committed:
Update General_TTS_inference_API_notebook_version2.ipynb

added colab import for ngrok token

Nanayeb34 committed:
Update General_TTS_inference_API_notebook_version2.ipynb

clean up of imports

wsedor commented on text-to-speech/General_TTS_inference_API_notebook_version2.ipynb:
```diff
@@ -172,8 +168,8 @@
       "source": [
-        "#Replace comment section below with your auth token\n",
-        "!ngrok config add-authtoken #place your access token here"
+        "token=userdata.get('ngrok_token')\n",
```
can we capitalize the token name and also the key so it's consistent with other notebooks. `"NGROK_TOKEN = userdata.get('NGROK_TOKEN')\n",`

Nanayeb34 committed:
Update General_TTS_inference_API_notebook_version2.ipynb

Nanayeb34 committed:
Delete text-to-speech/General_TTS_inference_API_notebook_version2.ipynb

Nanayeb34 committed:
Added Inference API notebook v2

added updated TTS inference notebook

Nanayeb34 committed:
Update README.md

updated ngrok portion to inform users to put their token into the secrets key section in colab.

Nanayeb34 committed:
Update General_TTS_inference_API_notebook_version2.ipynb

updated config path
